packages/@tinacms/toolkit/src/packages/form-builder/fields-builder.tsx

```diff
@@ -44,7 +44,12 @@ export function FieldsBuilder({ form, fields }: FieldsBuilderProps) {
     // @ts-ignore FIXME twind
     <FieldsGroup>
       {fields.map((field: Field) => (
-        <InnerField field={field} form={form} fieldPlugins={fieldPlugins} />
+        <InnerField
+          key={field.name}
```

Comment:
Glad to see this; I had the same change in my own PR (to clean up the `console`) 👍